config.go

```diff
@@ -88,13 +88,15 @@ import (
 type ProtocolVersion int
 
 const (
+	// ProtocolVersionMin is the minimum protocol version
 	ProtocolVersionMin ProtocolVersion = 0
-	ProtocolVersionMax                 = 3
+	// ProtocolVersionMax is the maximum protocol version
+	ProtocolVersionMax = 3
```

Comment:
This is nitpicky -
But I liked the _look_ of it where it was aligned with `ProtocolVersionMin` - can we revert it to how it was or is this one of the formatting errors you're trying to fix?